src/wp-includes/script-loader.php

```diff
@@ -108,7 +108,7 @@ function wp_default_packages_vendor( $scripts ) {
 	$vendor_scripts_versions = array(
 		'react'                       => '18.2.0',
 		'react-dom'                   => '18.2.0',
-		'regenerator-runtime'         => '0.14.0',
+		'regenerator-runtime'         => '0.13.11',
```

Comment:
@desrosj, flagging this one. I'm not sure why it's being downgraded, but we will want to revisit it.

Reply:
If I follow it correctly, it should be the version that is in the `package.json` as it's going to be used for caching purposes. It shouldn't get changed then.

Reply:
Good catch. Seems to be coming from https://github.com/WordPress/wordpress-develop/pull/5262/commits/d1960cc6051e07826bbb6bc3d2d13d0932b21a17.

Reply:
Updated with https://core.trac.wordpress.org/changeset/56739.